diff --git a/node_modules/react-native-gesture-handler/android/src/main/jni/cpp-adapter.cpp b/node_modules/react-native-gesture-handler/android/src/main/jni/cpp-adapter.cpp
index 7b5cbe4..18756fd 100644
--- a/node_modules/react-native-gesture-handler/android/src/main/jni/cpp-adapter.cpp
+++ b/node_modules/react-native-gesture-handler/android/src/main/jni/cpp-adapter.cpp
@@ -1,6 +1,7 @@
 #include <jni.h>
 #include <jsi/jsi.h>
 
+#include <react/renderer/bridging/bridging.h>
 #include <react/renderer/uimanager/primitives.h>
 
 using namespace facebook;
@@ -19,12 +20,8 @@ void decorateRuntime(jsi::Runtime &runtime) {
                     return jsi::Value::null();
                 }
 
-#if REACT_NATIVE_MINOR_VERSION >= 81
                 auto shadowNode = Bridging<std::shared_ptr<const ShadowNode>>::fromJs(
                         runtime, arguments[0]);
-#else
-                auto shadowNode = shadowNodeFromValue(runtime, arguments[0]);
-#endif
 
                 bool isViewFlatteningDisabled = shadowNode->getTraits().check(
                         ShadowNodeTraits::FormsStackingContext);
diff --git a/node_modules/react-native-gesture-handler/apple/RNGestureHandlerModule.mm b/node_modules/react-native-gesture-handler/apple/RNGestureHandlerModule.mm
index 108ad2d..7b496e4 100644
--- a/node_modules/react-native-gesture-handler/apple/RNGestureHandlerModule.mm
+++ b/node_modules/react-native-gesture-handler/apple/RNGestureHandlerModule.mm
@@ -14,6 +14,7 @@
 #import <ReactCommon/CallInvoker.h>
 #import <ReactCommon/RCTTurboModule.h>
 
+#import <react/renderer/bridging/bridging.h>
 #import <react/renderer/components/text/ParagraphShadowNode.h>
 #import <react/renderer/components/text/TextShadowNode.h>
 #import <react/renderer/uimanager/primitives.h>
@@ -102,11 +103,7 @@ void decorateRuntime(jsi::Runtime &runtime)
           return jsi::Value::null();
         }
 
-#if REACT_NATIVE_MINOR_VERSION >= 81
         auto shadowNode = Bridging<std::shared_ptr<const ShadowNode>>::fromJs(runtime, arguments[0]);
-#else
-        auto shadowNode = shadowNodeFromValue(runtime, arguments[0]);
-#endif
 
         if (dynamic_pointer_cast<const ParagraphShadowNode>(shadowNode)) {
           return jsi::Value(true);
